docs/sources/alerting/fundamentals/queries-conditions/_index.md

```diff
@@ -0,0 +1,98 @@
+In Alerting, you define a query to get the data you want to measure and a condition that needs to be met before an alert rule fires.
+
+An alert rule consists of one or more queries and expressions that select the data you want to measure.
+
+For more information on queries and expressions, see [Query and transform data]({{< relref "../../panels-visualizations/query-transform-data" >}})
+
+## Data source queries
+
+Queries in Grafana can be applied in various ways, depending on the data source and query language being used. Each data source’s query editor provides a customized user interface that helps you write queries that take advantage of its unique capabilities.
+
+Because of the differences between query languages, each data source query editor looks and functions differently. Depending on your data source, the query editor might provide auto-completion features, metric names, variable suggestions, or a visual query-building interface.
+
+Some common types of query components include:
+
+**Metrics or data fields**: Specify the specific metrics or data fields you want to retrieve, such as CPU usage, network traffic, or sensor readings.
+
+**Time range**: Define the time range for which you want to fetch data, such as the last hour, a specific day, or a custom time range.
+
+**Filters**: Apply filters to narrow down the data based on specific criteria, such as filtering data by a specific tag, host, or application.
+
+**Aggregations**: Perform aggregations on the data to calculate metrics like averages, sums, or counts over a given time period.
+
+**Grouping**: Group the data by specific dimensions or tags to create aggregated views or breakdowns.
+
+**Note**:
+
+Grafana does not support alert queries with template variables. More information is available [here](https://community.grafana.com/t/template-variables-are-not-supported-in-alert-queries-while-setting-up-alert/2514).
+
+## Expression queries
+
+In Grafana, an expression is used to perform calculations, transformations, or aggregations on the data source queried data. It allows you to create custom metrics or modify existing metrics based on mathematical operations, functions, or logical expressions.
+
+By leveraging expression queries, users can perform tasks such as calculating the percentage change between two values, applying functions like logarithmic or trigonometric functions, aggregating data over specific time ranges or dimensions, and implementing conditional logic to handle different scenarios.
+
+In Alerting, you can only use expressions for Grafana-managed alert rules. For each expression, you can choose from the math, reduce, and resample expressions. These are called multi-dimensional rules, because they generate a separate alert for each series.
+
+You can also use classic condition, which creates an alert rule that triggers a single alert when its condition is met. As a result, Grafana sends only a single alert even when alert conditions are met for multiple series.
```

Comment:
Perhaps we should mention this also here, or remove the text about classic conditions as we don't really want people using them.

> Classic condition expression queries always produce one alert instance only, no matter how many time series meet the condition.
> Classic conditions exist mainly for compatibility reasons and should be avoided if possible.